tests/authController.test.js

```diff
@@ -0,0 +1,49 @@
+const authController = require('../server/controllers/authController');
+const User = require('../server/models/usermodel');
+const httpMocks = require('node-mocks-http');
+const fakeUser = require('./mock-data/new-user.json');
+const validator = require('validator');
+
+User.create = jest.fn();
+
+let req, res, next;
+
+beforeEach(() => {
+    req = httpMocks.createRequest();
+    res = httpMocks.createResponse();
+    next = null;
+});
+
+describe("authController should a a new user to the database", () => {
```

Comment:
there's a typo here

Reply:
You might want to modify your test cases

e.g

`describe('Auth Controller API", () => {...`